src/test/e2e-emulated/debug.spec.ts

```diff
@@ -35,21 +35,29 @@ describe("debug", () => {
             "src/test/e2e-emulated/contracts/debug.tact",
         );
 
-        expect(debugLogs).toStrictEqual(`File ${filePath}:10:9
+        expect(debugLogs).toStrictEqual(`File ${filePath}:10:9:
+dumpStack();
 stack(2 values) : 10000000000 () 
-File ${filePath}:11:9
+File ${filePath}:11:9:
+dump("Hello world!");
 Hello world!
-File ${filePath}:12:9
+File ${filePath}:12:9:
+dump(123);
 123
-File ${filePath}:13:9
+File ${filePath}:13:9:
+dump(true);
 true
-File ${filePath}:14:9
+File ${filePath}:14:9:
+dump(false);
 false
-File ${filePath}:15:9
+File ${filePath}:15:9:
+dump(null);
 null
-File ${filePath}:16:9
+File ${filePath}:16:9:
+dump(myAddress());
 ${contract.address.toString({ bounceable: true })}
-File ${filePath}:17:9
+File ${filePath}:17:9:
+dump(newAddress(0, 0x83dfd552e63729b472fcbcc8c45ebcc6691702558b68ec7527e1ba403a0f31a8));
```

Comment:
btw, let's add some comments to the corresponding contract to make sure in cases like

```
dump(42);   // dumping the Answer to the Ultimate Question of Life, the Universe, and Everything
```

we don't actually get the comment part in the debug output

Reply:
But comments are included in the line.
Maybe changing the behaviur to only print the argument expression will address the original problem better? Like this:
```
File ${filePath}:17:9:
newAddress(0, 0x83dfd552e63729b472fcbcc8c45ebcc6691702558b68ec7527e1ba403a0f31a8)
EQCD39VS5jcptHL8vMjEXrzGaRcCVYto7HUn4bpAOg8xqB2N
```

Reply:
we can add "dump(" and ")" around the expression to give more visual context, but yes, it's not about the line per se, but about the dumped expression